api/team.go

```diff
@@ -456,6 +456,15 @@ func inviteMembers(c *Context, w http.ResponseWriter, r *http.Request) {
 		user = result.Data.(*model.User)
 	}
 
+	var invNum int64 = 0
+	for i, invite := range invites.Invites {
+		if result := <-Srv.Store.User().GetByEmail(c.Session.TeamId, invite["email"]); result.Err == nil || result.Err.Message != "We couldn't find the existing account" {
```

Comment:
Thanks for the feedback. I've filed an issue in our internal feature tracker. This is done in other places in the code and we should probably move to error code responses for localization.